docs/docs/guide/nextjs-13-and-growthbook.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: GrowthBook and Next.js 13 (beta)
+description: This guide walks you through using GrowthBook with Next.js 13 (beta).
+sidebar_label: Next.js 13 (beta)
+slug: nextjs-13-and-growthbook
+---
+
+# GrowthBook and Next.js 13 (beta)
+
+This document is a guide on how to add GrowthBook feature flags to your Next.js application. It assumes you are starting from scratch, and are using the new 'app directory' pattern introduced in Next 13.
+
+If you already have a NextJS application, you can skip to step 2.
+
+## 1. Create your Next.js App
```

Comment:
Stylistically I wouldn't number the headings:

- it's inconsistent with existing style
- when you add a new section in between other sections, you'll have to update the numbers and all references to any numbers that have changed. I'd link to the sub-section using an anchor instead. Heading ID's should be generated for all headings by Docusaurus.

Reply:
Going w/ separate section approach